package.json

```diff
@@ -41,7 +41,7 @@
   "dependencies": {
     "@gnosis.pm/dex-contracts": "^0.3.2",
     "@gnosis.pm/owl-token": "^3.1.0",
-    "@gnosis.pm/safe-contracts": "github:gnosis/safe-contracts#v1.1.1-dev.1",
+    "@gnosis.pm/safe-contracts": "v1.1.1-dev.2",
```

Comment:
If we are changing the contract here, will this have an impact on "verifying old setups"?

Reply:
Yes, this is a valid concern, if the contract changes in any non-trivial way then our code is broken. In this case, the changes didn't modify the bytecode.